incubator/hnc/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kubernetes-sigs/multi-tenancy/incubator/hnc
 
-go 1.12
+go 1.13
```

Comment:
General question - are we using the latest possible version of everything in this file?

Reply:
The ones we updated in this PR were the latest version.